log/format.go

```diff
@@ -262,6 +269,12 @@ func formatLogfmtValue(value interface{}) string {
 		// expensive.
 		return t.Format(timeFormat)
 	}
+	if term {
+		if s, ok := value.(TerminalStringer); ok {
+			// Custom terminal stringer provided, use that
+			return escapeString(s.TerminalString())
+		}
+	}
```

Comment:
Format shared is also used in JSON. I don't want to have the terminal stuff running for json logs.